renderer/components/Editor.tsx

```diff
@@ -81,6 +84,9 @@ interface State {
   simulate: boolean;
   isRunning: boolean;
   fontsize?: number;
+  keyboardControl: boolean;
+  bitmap: number;
+  buttonStyle: string;
```

Comment:
Just to be a bit clearer, I think we can rename these variables to:
`keyboardControl` -> `isKeyboardModeToggled`
`bitmap` -> `keyboardBitmap`

Also since `buttonStyle` derives its value from `keyboardControl` we can probably do an inline ternary operation instead and remove `buttonStyle` from the state, will explain more below where the style is set.

Reply:
We can remove `buttonStyle` too